katz/src/main/kotlin/katz/effects/Duration.kt

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
We should add attributions in the Copying or License files but I thought we decided not to have headers copyrights.